test/modules/sungeun/init/printModuleInitOrder.good

```diff
@@ -3,7 +3,6 @@ Initializing Modules:
     OwnedObject
     Atomics
      MemConsistency
-    NetworkAtomics
```

Comment:
Why is this removed here?

Reply:
Ah thanks for catching that, probably I copied the exec.out.tmp to wrong good file.